spec/commands/outdated_spec.rb

```diff
@@ -244,6 +246,7 @@
   shared_examples_for "version update is detected" do
     it "reports that a gem has a newer version" do
       subject
+      expect(out).to_not include("ERROR REPORT TEMPLATE")
```

Comment:
Hm, I was following precedence here (https://github.com/bundler/bundler/blob/master/spec/bundler/friendly_errors_spec.rb#L29 and https://github.com/bundler/bundler/blob/master/spec/install/gemfile/path_spec.rb#L161). There isn't any consistent output that would be generated after the `Outdated gems included in the bundle:` portion that can be relied on.